src/test/java/com/stripe/model/LiveStripeResponseGetterTest.java

```diff
@@ -0,0 +1,62 @@
+package com.stripe.net;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Account;
+import com.stripe.net.LiveStripeResponseGetter;
+import com.stripe.net.RequestOptions;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class LiveStripeResponseGetterTest {
+	LiveStripeResponseGetter srg = new LiveStripeResponseGetter();
```

Comment:
Can you make this part of a `@Before` (or something)? Otherwise, this test is stateful.